doc/source/how-to-configure-clients.rst

```diff
@@ -1,17 +1,16 @@
-Configure clients
+Configure Clients
 =================
 
-Along with model parameters, Flower can send configuration values to clients.
-Configuration values can be used for various purposes. They are, for example, a popular
-way to control client-side hyperparameters from the server.
+Flower provides the ability to send configuration values to clients, allowing
+server-side control over client behavior. This feature enables flexible and dynamic
+adjustment of client-side hyperparameters, improving collaboration and experimentation.
 
 Configuration values
```

Comment:
I'm thinking this whole section should better become a sub-section in the `Configuration through built-in strategies` section so there is less chance of confusion with other configs we have (e.g. the Run Config). I'd therefore present all this content as "FitConfig" and "EvaluateConfig" specifics.

Reply:
likely, since the next section shows a semi-complete example on how to create a `fit_config` callback and how to later "read" those values in the `fit()` method of a client, we can remove the code blocks that present these dictionaries w/o being part of the call back. Showing the `JSON` serialization bit directly in the `fit_config()` callback would be :100:, it's great to have it included in the docs!